doc/api/console.markdown

```diff
@@ -9,8 +9,8 @@ The module exports two specific components:
 
 * A `Console` class with methods such as `console.log()`, `console.error()` and
   `console.warn()` that can be used to write to any Node.js stream.
-* A global `console` instance configured to write to `stdout` and `stderr`.
-  Because this object is global, it can be used without calling
+* A global `console` instance configured to write to process `stdout` and
```

Comment:
Maybe this should instead be:

<pre>
... to write to `process.stdout` and `process.stderr`.
</pre>